src/Microsoft.Framework.Runtime/GlobalSettings.cs

```diff
@@ -53,7 +53,7 @@ public static bool TryGetGlobalSettings(string path, out GlobalSettings globalSe
                                              jobject.ValueAsStringArray("sources") ??
                                              new string[] { };
 
-                    globalSettings.ProjectSearchPaths = projectSearchPaths;
+                    globalSettings.ProjectSearchPaths = new List<string>(projectSearchPaths);
```

Comment:
nit: I've seen `ToList` used in most places to do this. But I don't really care much either way :)